src/styles/main.scss

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0;
+}
+
+body {
+  display: flex;
+  height: 100vh;
+}
+
+.minute {
+  position: absolute;
+  top: 25%;
+  left: 49%;
+  width: 9px;
+  height: 150px;
+  background-color: #004;
+  animation: min 3600s linear infinite;
+  border-radius: 50%;
+  box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.4);
+}
+
+.second {
+  box-sizing: border-box;
+  position: absolute;
+  top: 17%;
+  left: 49%;
+  width: 8px;
+  height: 200px;
+  background-color: red;
+  animation: sec 60s linear infinite;
+  border-radius: 50%;
+  box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.4);
+}
+
+.clock {
+  margin: auto;
+  box-sizing: border-box;
+  width: 430px;
+  height: 430px;
+  border-radius: 100%;
+  background-color: white;
+  border: 1px thick double #004;
+  box-shadow: 0 0 8px 0 rgba(0, 0, 0, 0.8);
+}
+
+@keyframes sec {
```

Comment:
they are the same, u can delete one of them and nothing will change

Reply:
no, there is still two keyframes

Reply:
sorry, have misunderstood you comment, now definitely it is fixed )